src/components/FollowersFollowing.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import { Button } from "antd";
+import FollowersFollowingUsers from "./FollowersFollowingUsers";
+import { preFabProfiles } from "../test/testProfiles";
+
+const FollowersFollowing = (): JSX.Element => {
+  const [selectedListTitle, setSelectedListTitle] = useState<string>("");
+
+  const tempUserList = [
+    {
+      profile: preFabProfiles[0],
+      icon: preFabProfiles[0].icon,
+      name: preFabProfiles[0].name,
+      following: true,
+      followsMe: true,
+    },
+    {
+      profile: preFabProfiles[1],
+      icon: preFabProfiles[1].icon,
+      name: preFabProfiles[1].name,
+      following: true,
+      followsMe: false,
+    },
+    {
+      profile: preFabProfiles[2],
+      icon: preFabProfiles[2].icon,
+      name: preFabProfiles[2].name,
+      following: false,
+      followsMe: true,
+    },
+  ];
+
+  const handleClick = (listTitle: string) => {
+    if (selectedListTitle === listTitle) {
+      setSelectedListTitle("");
+      return;
+    }
+    if (listTitle === "showFollowers") {
+      setSelectedListTitle(listTitle);
+    } else {
+      setSelectedListTitle(listTitle);
+    }
+  };
```

Comment:
This is for a toggle right?

```suggestion
  const toggleSelectedList = (listTitle: string) => {
    if (selectedListTitle === listTitle) setSelectedListTitle("");
    else setSelectedListTitle(listTitle);
  };
```

Reply:
duhh!! thanks for catching that